src/api/app/controllers/webui/architectures_controller.rb

```diff
@@ -0,0 +1,30 @@
+# Enabling/Disabling default architectures
+class Webui::ArchitecturesController < Webui::WebuiController
+  before_filter :require_admin
+
+  def index
+    @architectures = Architecture.order(:name)
+  end
+
+  def bulk_update_availability
+    archs = params[:archs] || []
+    all_valid = true
+    archs.each do |archname, value|
+      available = value == '1'
+      if old = Architecture.where(name: archname, available: !available).first
```

Comment:
yeah me too. we've copied this code over from the old [configuration controller](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/configuration_controller.rb#L63) to avoid going down another rabbit hole ;-)